index.js

```diff
@@ -573,6 +601,62 @@ class FalconxClient {
     return this.makeHTTPRequest('/trade_sizes', 'get');
   }
 
+  /**
+     * Get derivative trade information with current mtm data.
+     * @param {String} trade_status {'open'|'closed'|'settled'|'defaulted'}
+     * @param {String} product_type {'ndf'|'call_option'|'put_option'|'irs'|'option'}
+     * @param {String} market_list: comma separated 'BTC-USD,ETH-USD'
+     * @returns {Promise<DerivativeTrade[]|FxError>}
+     * @example {
+        'status': 'success',
```

Comment:
if you need the status column then you'll have to define a top level typedef. Something like `DerivativeTradeResponse` with one of the properties as status and another as `@property {DerivativeTrade[]} response`